### PR TITLE
vcenter alerts

### PR DESCRIPTION
**Pull request by bhowmickatish**
### What does this PR do?
https://jira.nutanix.com/browse/ENG-304723

A brief description of the change being made with this pull request.
https://jira.nutanix.com/browse/ENG-304723

### Motivation
raise vcenter alerts
handle runtime faults of vcenter apis

### CODE REVIEW

File: vsphere/datadog_checks/vsphere/vsphere.py [hunk 1]
```diff
@@ -158,6 +172,11 @@ def __init__(self, name, init_config, agentConfig, instances):
         self.metric_ids = {}
         self.latest_event_query = {}
 
+        #init the statsd client
+        statsd_host = init_config.get('statsd_server_host',STATSD_SERVER_HOST)
```
Comment (ahalam) [inline vsphere/datadog_checks/vsphere/vsphere.py:176]: Where do we specify this?

**Review by ahalam — APPROVED** [review]

File: vsphere/datadog_checks/vsphere/vsphere.py (hunk 1 above)
Comment (bhowmickatish) [reply vsphere/datadog_checks/vsphere/vsphere.py:176]: Vsphere.yaml init section like rest of configuration parameters , else it uses default values 127.0.0.1 and 9000

Comment (lattwood) [inline vsphere/datadog_checks/vsphere/vsphere.py:176]: I think you missed a space after the comma.

File: vsphere/datadog_checks/vsphere/vsphere.py [hunk 2]
```diff
@@ -158,6 +172,11 @@ def __init__(self, name, init_config, agentConfig, instances):
...
+        statsd_port = init_config.get('statsd_server_port',STATSD_SERVER_PORT)
```
Comment (lattwood) [inline vsphere/datadog_checks/vsphere/vsphere.py:177]: I think you missed a space after the comma.

**Pull request merged**